style.css

```diff
@@ -0,0 +1,186 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@1&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+body {
+  position: relative;
+  width: 375px;
+  height: 6968px;
+  margin: auto;
+}
+
```

Comment:
- [ ] [OPTIONAL] Kindly remember that you are styling a mobile version. You can open the developer tools in your browser and select the 'Responsive Design Mode' to better view how your app looks in the mobile view. It's not best practice to set a fixed width or height in these cases.

Reply:
@leonardodiasb Thank you for reviewing my project, I will make the necessary corrections as suggested